charts/kubeflow/templates/katib/user-roles/clusterrole.kubeflow-katib-admin.yaml

```diff
@@ -1,11 +1,11 @@
 apiVersion: rbac.authorization.k8s.io/v1
 kind: ClusterRole
 metadata:
-  name: kubeflow-katib-admin
   labels:
-    rbac.authorization.kubeflow.org/aggregate-to-kubeflow-admin: "true"
+    {{- include "kubeflow.kubeflowRoles.kubeflowAdminRoleLabel" . | nindent 4 }}
+  name: {{ include "kubeflow.katib.kfNbAdminClusterRoleName" . }}
```

Comment:
Maybe let's rename to `kubeflow.katib.adminClusterRoleName`.

Reply:
Updated